RNTester/Podfile

```diff
@@ -24,3 +27,24 @@ target 'RNTester' do
   # To use fabric: add `fabric_enabled` option to the use_react_native method above, like below
   # use_react_native!(path: "..", turbo_modules_enabled: true, fabric_enabled: true)
 end
+
+target 'RNTester-macOS' do
+  platform :osx, '10.14'
+  # Uncomment for Swift
+  # use_frameworks!
+
+  project 'RNTesterPods.xcodeproj'
+  
+  # Enable TurboModule
+  use_react_native!(path: "..", turbo_modules_enabled: true)
+  # pod 'React-turbomodule-samples', :path => '../ReactCommon/turbomodule/samples'
+
+  # Additional Pods which aren't included in the default Podfile
+  pod 'React-ART', :path => '../Libraries/ART'
+  pod 'React-RCTPushNotification', :path => '../Libraries/PushNotificationIOS'
```

Comment:
Does macOS need this?

Reply:
Aye. While there’s `iOS` in the directory name, it’s currently only a matter of naming. For instance, [here’s](https://github.com/microsoft/react-native/blob/master/Libraries/PushNotificationIOS/RCTPushNotificationManager.m#L172-L191) some macOS specific code. Perhaps in the future this lib needs renaming, but I guess that will also depend on how macOS specific code will be kept out of core.